docs/tutorial.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "pori_graphkb_python-tutorial.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "authorship_tag": "ABX9TyPsZefAkXO6sZhzyUDWur8c",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "pythonjvsc74a57bd04a7f6883d8b4c4ea9250c869e6fc6033cc88c7f8e8bca79e110c33b1152823f7",
+      "display_name": "Python 3.8.3 64-bit"
+    },
+    "language_info": {
+      "name": "python",
+      "version": "3.8.3"
+    },
+    "metadata": {
+      "interpreter": {
+        "hash": "4a7f6883d8b4c4ea9250c869e6fc6033cc88c7f8e8bca79e110c33b1152823f7"
+      }
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rP6xlIsD79xj"
+      },
+      "source": [
+        "# GraphKB Variant Matching Tutorial\n",
+        "\n",
+        "This tutorial is an interactive notebook which can be run using ([google colab](https://colab.research.google.com/github/bcgsc/pori_graphkb_python/blob/master/docs/pori_graphkb_python_tutorial.ipynb)) or a local jupyter server (**reccommended** if matching patient data). This tutorial will cover basic matching of variants using the python GraphKB adaptor against an instance of the GraphKB API. \n",
+        "\n",
+        "Users must first have login credentials to an instance of GraphKB API (or use the demo server). Note for users using the demo credentials and server, the data is limited and more complete annotations would be expected for a production instance of GraphKB.\n",
+        "\n",
+        "For the purposes of this tutorial we will be matching the known KRAS variant `p.G12D` to the demo instance of GraphKB. You can adjust the API instance by changing the setup variables below\n",
+        "\n",
+        "To run this locally download this file and start the server from the command line as follows\n",
+        "\n",
+        "```\n",
+        "jupyter notebook notebook.ipynb\n",
```

Comment:
Note - the notebook is called 'tutorial.ipynb', not 'notebook.ipynb'.  But I think jupyter seems to open a Home Page tree, instead of a specific notebook anyway.

Jupyter doesn't seem to be in the current setup.py.  Maybe add it to the DOC_REQS, or mention `pip install jupyter` here?